### PR TITLE
devise tasks

### PR DESCRIPTION
**Pull request by sCar3Cr0w25**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ default: &default
   encoding: utf8mb4
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: root
-  password: "Password1@#"
+  password: root
```
Comment (CW-AnhNH) [inline config/database.yml:17]: use ENV

File: test/controllers/pages_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```
Comment (CW-AnhNH) [inline test/controllers/pages_controller_test.rb:1]: remove unuse test folder, we will using rspec for testing

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class PagesController < ApplicationController
+  
+  def home
+    @users = User.where(admin: false).or(User.where(admin: nil))
```
Comment (CW-HauNC) [inline app/controllers/pages_controller.rb:4]:
```suggestion
    @users = User.where.not(admin: true)
```

File: app/models/user.rb [hunk 4]
```diff
@@ -1,2 +1,14 @@
 class User < ApplicationRecord
+  extend Devise::Models
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  
+  # enum role: [:user, :admin]
```
Comment (CW-HauNC) [inline app/models/user.rb:8]: remove if don't need

File: app/views/admin/dashboard/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Admin::Dashboard#index</h1>
+<p>Find me in app/views/admin/dashboard/index.html.erb</p>
```
Comment (CW-HauNC) [inline app/views/admin/dashboard/index.html.erb:2]: add end line

File: bin/bundle [hunk 6]
```diff
@@ -8,62 +8,64 @@
 # this file is here to facilitate running it.
```
Comment (CW-HauNC) [inline bin/bundle:8]: ignore change this file

File: bin/rails [hunk 7]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path("../config/application", __dir__)
```
Comment (CW-HauNC) [inline bin/rails:2]: ignore change

File: bin/rake [hunk 8]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
```
Comment (CW-HauNC) [inline bin/rake:1]: ignore change

File: bin/setup [hunk 9]
```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env ruby
-require "fileutils"
```
Comment (CW-HauNC) [inline bin/setup:2]: ignore change

File: config/boot.rb [hunk 10]
```diff
@@ -1,4 +1,6 @@
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
```
Comment (CW-HauNC) [inline config/boot.rb:1]: ignore change

File: config/environment.rb [hunk 11]
```diff
@@ -1,5 +1,7 @@
+# frozen_string_literal: true
+
 # Load the Rails application.
```
Comment (CW-HauNC) [inline config/environment.rb:3]: ignore change

File: config/environments/development.rb [hunk 12]
```diff
@@ -1,4 +1,6 @@
-require "active_support/core_ext/integer/time"
```
Comment (CW-HauNC) [inline config/environments/development.rb:1]: ignore change

File: config/environments/production.rb [hunk 13]
```diff
@@ -1,4 +1,6 @@
-require "active_support/core_ext/integer/time"
```
Comment (CW-HauNC) [inline config/environments/production.rb:1]: ignore change

**Pull request merged**